wd_helper.au3

```diff
@@ -2376,6 +2382,77 @@ Func _WD_GetElementByRegEx($sSession, $sMode, $sRegExPattern, $sRegExFlags = "",
 	Return SetError(__WD_Error($sFuncName, $iErr), $iRow, $vResult)
 EndFunc   ;==>_WD_GetElementByRegEx
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_Storage
+; Description ...:
+; Syntax ........: _WD_Storage($sSession,  $sKey[,  $vValue = Default[,  $nType = Default]])
+; Parameters ....: $sSession            - Session ID from _WD_CreateSession
+;                  $vKey                - Key to manipulate.
+;                  $vValue              - [optional] Value to store.
+;                  $nType               - [optional] Storage type. Default is $_WD_STORAGE_Local
+; Return values .: Success - Response from _WD_ExecuteScript() and sets @error to $_WD_ERROR_Success
+;                  Failure - @error set to $_WD_ERROR_InvalidArgue if there are no matches OR
+;                            Response from _WD_ExecuteScript() and sets @error to value returned from _WD_ExecuteScript()
+; Author ........: Danp2
+; Modified ......:
+; Remarks .......:	See below for special conditions --
+;
+;					| Parameter | Condition | Action                     |
+;					|-----------|-----------|----------------------------|
+;					| $vKey     | Numeric   | Return name of the Nth key |
+;					| $vKey     | Null      | Clear storage              |
+;					| $vValue   | Null      | Remove key from storage    |
```

Comment:
should here be added a case when $vValue is `Default` ?

Reply:
When $vValue is `Default`, then the value of the key will be retrieved from storage. This is "standard" behavior IMO. With the above table, I was attempting to document the "special" behaviors.

Reply:
Ok. Click as resolved.